### ISSUE TITLE
Its applicability to text data

### ISSUE THREAD

**Issue by HyunkuKwon** [Computational-Content-Analysis-2018/2-Feb-Maximum-Likelihood-Estimation-of-Observer-Error-rates-using-the-EM-Algorithm#7]
Text data are prone to typos, misspellings, and idiosyncratic usage of words, which makes researchers hard to parse and analyze such data. Can we apply such EM algorithms to infer irregularities in text data?

**Comment by sunnyJy** [comment]
Interesting!
In that sense, the "true response" becomes the "correct spelling words".
I wonder whether "normalization" could do this, by stemming?
